plugins/modules/ios_acl_interfaces.py

```diff
@@ -491,12 +490,8 @@
 
 - name: Parse the commands for provided configuration
   cisco.ios.ios_acl_interfaces:
-    running_config:
-      "interface GigabitEthernet0/1
-       ip access-group 110 in
-       ip access-group 123 out
-       ipv6 traffic-filter temp_v6 in
-       ipv6 traffic-filter test_v6 out"
+    running_config: interface GigabitEthernet0/1 ip access-group 110 in ip access-group
```

Comment:
Fwiw, this is kinda unreadable. I _think_ the collection prep script does this, because it doesn't specify a line-length while running black. We need to run it manually afterwards.